### PR TITLE
replace menu with adhoc sandbox

### PR DESCRIPTION
**Pull request by congyu-lu**

### CODE REVIEW

**Comment by xzhou82** [comment]
thanks, please show sample name in sandbox header, as in line 308 of sampletable.js
for both scatter and matrix

**Comment by congyu-lu** [comment]
> thanks, please show sample name in sandbox header, as in line 308 of sampletable.js for both scatter and matrix

Thanks, added sample names in sandbox header

File: client/plots/matrix.interactivity.js [hunk 1]
```diff
@@ -116,6 +117,7 @@ export function setInteractivity(self) {
 					.text('Disco plot')
```
Comment (xzhou82) [inline client/plots/matrix.interactivity.js:117]: line 113 `if()` should not be nested under singleSampleGenomeQuantification

Comment (congyu-lu) [reply client/plots/matrix.interactivity.js:122]: Thanks, fixed for both matrix and scatterplot

File: client/plots/matrix.interactivity.js [hunk 2]
```diff
@@ -84,38 +84,45 @@ export function setInteractivity(self) {
 	}
 
 	self.mouseclick = function(event) {
-		if (self.state.termdbConfig.queries?.singleSampleGenomeQuantification) {
+		if (
+			self.state.termdbConfig.queries?.singleSampleGenomeQuantification ||
+			self.state.termdbConfig.queries?.singleSampleMutation
+		) {
 			const sample = event.target.__data__
 			sample.sample_id = sample.row?.sampleName
```
Comment (xzhou82) [inline client/plots/matrix.interactivity.js:92]: this is not right

**Review by xzhou82 — APPROVED** [review]
thanks i now realize a matrix with lots of samples won't show sample names upon zooming in, thus preventing from clicking sample name. but this is a separate issue

**Pull request merged**